istresearch/rapidpro
# P4-3873: fix delete user button

# For the Reviewer

When this is complete, you should approve the PR via github.

# For the Reviewee

## Summary
Admin button to delete a user is fixed.

#### Release Note
Admins of the website can now use the fixed delete user button.

#### Breaking Changes
None.

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

1. Sign in as a superuser (site admin)
2. in the users list area, pick a user to delete
3. use the Delete button to delete the user
4. click OK at the confirmation dialog
5. verify user has been removed from the list of users
6. (optional) check the db for the removed user, their "is_active" should be False.
7.
